### PR TITLE
update output section

### PR DESCRIPTION
**Pull request by rhagarty**
- added sample output section
- moved current notebook (with output) to data/examples
- cleared all output from notebook
- minor README fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,9 +255,17 @@ There are several ways to execute the code cells in your notebook:
...
-> *Note:* To see the code and markdown cells without output, you can view [the raw notebook in the Github viewer](notebooks/elasticsearch-spark-recommender.ipynb).
+![](images/ratings-table.png)
+
+![](images/genre-table.png)
+
+After loading our model into Solr, we can predict which movies people will like based on ratings of similar movies in the same genre.
+
+![](images/movie-list.png)
+
+The example output in the `data/examples` folder shows the output of the notebook after running it in full. View it it [here](data/examples/solr-hdp-spark-recommender.ipynb).
```
Comment (stevemar) [inline README.md]: you can link data/examples...

```
[`data/examples`](data/examples)
```

Comment (stevemar) [reply README.md]: Also, last line, "it" twice.

**Comment by rhagarty** [comment]
resolved Steve's issues

**Review by markstur — COMMENTED** [review]
Source notebook changed from py2 to py3.  Was that intentional/OK?

The example output looks good. Github seems to know it doesn't need nbviewer for this one.

**Review by stevemar — APPROVED** [review]

**Review by dilipbiswal — APPROVED** [review]
LGTM

**Comment by rhagarty** [comment]
Set notebook back to using py2
Thanks @markstur

**Review by stevemar — APPROVED** [review]

**Pull request merged**